memcached-operator/pkg/apis/cache/v1alpha1/memcached_types.go

```diff
@@ -24,7 +25,9 @@ type MemcachedStatus struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 
-	// Status of Nodes
+	// Nodes are the names of the memcached pods
+	// +kubebuilder:validation:UniqueItems=true
+	// +listType=set  //More Info: https://godoc.org/k8s.io/kube-openapi/pkg/idl
```

Comment:
Move the `//More info` comment, as described here: https://github.com/operator-framework/operator-sdk/pull/2130#discussion_r340929945